chipsec/options.py

```diff
@@ -0,0 +1,53 @@
+# CHIPSEC: Platform Security Assessment Framework
+# Copyright (c) 2023, Intel Corporation
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; Version 2.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# Contact information:
+# chipsec@intel.com
+#
+
+import os
+import yaml

```

Comment:
Open question: Do we want to use YAML and introduce a new dependency or do we want to use JSON (or similar) which we already use other places and don't need to introduce a new dependency?  
@BrentHoltsclaw  @frinzell

Reply:
YAML is much easier to read and edit than JSON.  If there is a concern about adding a dependency then should probably look into using a configuration file with configparser

Reply:
Small YAML files are fine for reading, larger ones not so much.  The indention-based nesting is the worst, IMO.  Would rather use TOML.  But JSON or even XML would be preferred over YAML.